src/Sulu/Bundle/RouteBundle/DependencyInjection/RouteGeneratorCompilerPass.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\RouteBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Collects route-generator by configured mappings.
+ */
+class RouteGeneratorCompilerPass implements CompilerPassInterface
+{
+    const TAG_NAME = 'sulu.route_generator';
+    const SERVICE_ID = 'sulu_route.manager.route_manager';
+    const PARAMETER_NAME = 'sulu_route.mappings';
```

Comment:
no the validation of the options will also be done in this class after changing to alias.